lib/tenios/api/record_call.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Tenios
+  module API
+    class RecordCall
+      attr_reader :client
+
+      def initialize(client)
+        @client = client
+      end
+
+      def start(call_uuid:)
+        client.http_client.post(
+          '/record-call/start',
+          access_key: client.access_key,
+          call_uuid: call_uuid
+        ).body
+      end
+
+      def stop(recording_uuid:, call_uuid:)
```

Comment:
Only the `recording_uuid` is necessary, no? 🤔

Reply:
The docs are a bit confusing.

The example table includes only `access_key` and `call_uuid` as parameters, but the example JSON payload includes the `recording_uuid` as well.

I'll do some manual testing to double check

Reply:
Looks like both `call_uuid` and `recording_uuid` are both required, seeing 400 responses if I omit either.